python/tank/util/shotgun.py

```diff
@@ -398,7 +398,7 @@ def create_sg_app_store_connection():
     return g_app_store_connection
 
 
-def __get_app_store_connection_information():
+def _get_app_store_connection_information():
```

Comment:
Just curious: What's the difference here between `_get_app_store_connection_information` (one underscore) and `__get_app_store_proxy_setting` (two underscores)? These are just methods outside of a class, so no protected/private distinction. Could/should we keep it consistent? One underscore? I think in other places in the code, we use single underscore unless you have reasons to use two (e.g. you really don't want a deriving class to be able to subclass your methods)